infrastructure/logic-app/rota/parameters-prod.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "environmentName":{
+            "value": "stg"
+        },
+        "workflows_hmidturota_name": {
+            "value": "hmi-la-dturota-stg"
+        },
+        "storageAccountName":{
+            "value": "hmidtustg"
+        },
+        "storageAccessKey":{
+            "reference": {
+                "keyVault": {
+                   "id": "/subscriptions/3eec5bde-7feb-4566-bfb6-805df6e10b90/resourceGroups/hmi-sharedservices-stg-rg/providers/Microsoft.KeyVault/vaults/hmi-shared-kv-stg"
+                },
+                "secretName": "dtu-storage-account-key"
+             }
+        },
+        "appId":{
+            "value": "#{keyVaultUserId}#"
+        },
+        "appSecret":{
+            "value": "#{keyVaultUserSecret}#"
+        },
+        "tenantId":{
+            "value": "#{azureTenant}#"
+        },
+        "keyvaultName":{
+            "value": "hmi-shared-kv-stg"
+        },
+        "hmiSchedulesUrl":{
+            "value": "https://hmi-apim.stg.platform.hmcts.net/hmi/schedules"
+        },
+        "hmiScopeUrl":{
+            "value": "api://d49d7fa2-6904-4223-a907-3b8992af4190"
```

Comment:
Would these not be better coming from Keyvault rather than hardcoded into our public code?

Reply:
this is not sensitive information, so no need to place in KV

Reply:
Ok thanks for checking @johnoneill46 .